flutter_appauth_platform_interface/analysis_options.yaml

```diff
@@ -0,0 +1,191 @@
+# Specify analysis options.
```

Comment:
Thanks but it's not necessary to add this file here. I originally left this out due to issues back many years ago. It should be possible to just have this at the root of the repository. As you're trying to fix the linter, would you be able to try fix this up putting it at the root and remove all other instances of the file?

Reply:
Done

Reply:
Could you help remove the other instances of analysis_options.yaml whilst you're at it?

Reply:
It's done, i remove in flutter_appauth, example and flutter_appauth_platform_interface.
and move in root of repository